cmd/revision.go

```diff
@@ -69,6 +70,7 @@ func revisionCmd() *cobra.Command {
 
 	revisionCmd.Flags().BoolP("suppress-secrets", "q", false, "suppress secrets in the output")
 	revisionCmd.Flags().StringArrayVar(&diff.suppressedKinds, "suppress", []string{}, "allows suppression of the values listed in the diff output")
+	revisionCmd.Flags().IntVarP(&diff.outputContext, "context", "C", -1, "output NUM lines of context around changes")
```

Comment:
For anyone interested - this corresponds to the equivalent option in the `diff` command:

```console
$ diff --help | grep -- -C
  -c  -C NUM  --context[=NUM]  Output NUM (default 3) lines of copied context.
```